contrib/templates/data-engineering/assets/etl-pipeline/template/assets/{{.pipeline_name}}/{{.pipeline_name}}.pipeline.yml.tmpl

```diff
@@ -0,0 +1,17 @@
+resources:
+  pipelines:
+    {{.pipeline_name}}:
+      name: {{.pipeline_name}}
+      serverless: true
+      {{- if or (eq default_catalog "") (eq default_catalog "hive_metastore")}}
+      ## Specify the 'catalog' field to configure this pipeline to make use of Unity Catalog:
+      # catalog: catalog_name
+      {{- else}}
+      catalog: {{default_catalog}}
+      {{- end}}
+      target: {{.pipeline_name}}_${bundle.environment}
```

Comment:
will this schema be created on demand?

Reply:
This is actually an error: it should say `${vars.schema}`. And yes, it's created on demand by the pipeline.

The above should use `${vars.catalog}`.